src/main/java/org/takes/http/BkBasic.java

```diff
@@ -70,16 +84,25 @@ public BkBasic(final Take tks) {
         this.take = tks;
     }
 
+    /**
+     * Read from socket.
+     * @param socket TCP socket with HTTP
+     * @throws IOException if some problem occurs
+     * @todo #306 15min/DEV : the server should use same socket for new coming
+     *  requests if the connection is persistent.
+     *  And the server should create new socket for new coming requests if the
+     *  connection is not persistent
```

Comment:
@mstzn36 it's not clear what do you want me, a developer working with this puzzle, to do. You're just stating the fact about something... But I can't understand what needs to be done..